pkg/ebpf/tracee.go

```diff
@@ -384,7 +384,7 @@ func (t *Tracee) Init() error {
 	// Init kernel symbols map
 
 	if initReq.kallsyms {
-		err = t.UpdateKernelSymbols()
+		err = t.NewKernelSymbols()
```

Comment:
I'd like to move to the Lazy symbols in this PR. I've ran our internal e2e on it before so it should be good to go.